pkg/process/collector.go

```diff
@@ -0,0 +1,121 @@
+package process
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs"
```

Comment:
This dependency is only used for “/proc” const string. As “Go proverbs” go, “A little copying is better than a little dependency”.

Reply:
Good point. Done.